JS_MODULE@0..27
  LIST@0..27
    JS_THROW_STATEMENT@0..5
      None THROW_KW@0..5 "throw" None
    JS_EXPRESSION_STATEMENT@5..27
      NEW_EXPR@5..27
        Whitespace(1) NEW_KW@6..9 "new" Whitespace(1)
        NAME_REF@10..15
          None IDENT@10..15 "Error" None
        ARG_LIST@15..27
          None L_PAREN@15..16 "(" None
          LIST@16..26
            LITERAL@16..26
              None STRING@16..26 "\"oh no :(\"" None
          None R_PAREN@26..27 ")" None
--
error[SyntaxError]: Linebreaks between a throw statement and the error to be thrown are not allowed
  ┌─ throw_stmt_err.js:2:1
  │
2 │ new Error("oh no :(")
  │ ^^^
  │ │
  │ Help: did you mean to throw this?
  │ A linebreak is not allowed here

--
throw
new Error("oh no :(")
